[OSDOCS-3505] ShiftStack NFV UX improvements (#46276)
openshift/openshift-docs: modules/installation-openstack-ovs-dpdk-requirements.adoc

// Module included in the following assemblies:
//
// * installing/installing_openstack/installing-openstack-nfv-preparing

> :_content-type: PROCEDURE

Reviewer: This isn't a procedure.

Author: ```suggestion
:_content-type: REFERENCE
```

[id="installation-openstack-ovs-dpdk-requirements_{context}"]
= Requirements for clusters on {rh-openstack} that use OVS-DPDK

To use Open vSwitch with the Data Plane Development Kit (OVS-DPDK) with your deployment, you must meet the following requirements:

> * Plan your RHOSP OVS-DPDK deployment by referring to link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/network_functions_virtualization_planning_and_configuration_guide/assembly_ovsdpdk_parameters[Planning your OVS-DPDK deployment] in the Network Functions Virtualization Planning and Configuration Guide.

Author: ```suggestion
* Plan your {rh-openstack-first} OVS-DPDK deployment by referring to link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/network_functions_virtualization_planning_and_configuration_guide/assembly_ovsdpdk_parameters[Planning your OVS-DPDK deployment] in the Network Functions Virtualization Planning and Configuration Guide.
```

> * Configure your RHOSP OVS-DPDK deployment according to link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/network_functions_virtualization_planning_and_configuration_guide/part-dpdk-configure[Configuring an OVS-DPDK deployment] in the Network Functions Virtualization Planning and Configuration Guide.

Author: ```suggestion
* Configure your {rh-openstack} OVS-DPDK deployment according to link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/network_functions_virtualization_planning_and_configuration_guide/part-dpdk-configure[Configuring an OVS-DPDK deployment] in the Network Functions Virtualization Planning and Configuration Guide.
```
